tests/main.cpp

```diff
@@ -8,20 +9,28 @@
 #include <stack>
 
 #include <GLFW/glfw3.h>
+#include "../../tangram-es/core/include/glm/glm/glm.hpp"
```

Comment:
@hjanetzek we can also use graphics flow for native discussions. Thats what Brett meant when he recommended not having a separate flow for native.